.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,130 @@
+
+    name: ${{ matrix.label }}
+    runs-on: ${{ matrix.os }}
+    if: "!contains(github.event.head_commit.message, 'bench')"
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: atharva-2001/tardis
+          ref: benchmark_base
+
+      - name: Download Atom Data
+        run: |
+          wget https://raw.githubusercontent.com/tardis-sn/tardis/83cd1f13088e95e087c97370af04bda6cf9956b7/.ci-helpers/download_reference_data.sh
+          bash download_reference_data.sh atom_data/kurucz_cd23_chianti_H_He.h5
+          mv ./tardis-refdata/atom_data/kurucz_cd23_chianti_H_He.h5  benchmarks/data/
+        
+      - name: Install asv
+        run: pip install asv
+      
+      - name: Run benchmarks for last 5 commits if not PR
+        # if: github.event_name != 'pull_request'
+        run: |
+          git log -n 5  --pretty=format:"%H" >> tag_commits.txt
+          asv machine --yes
+          asv run HASHFILE:tag_commits.txt -e | tee asv-output.log  
+          if grep -q Traceback asv-output.log; then 
+            echo "Some benchmarks have errors!"
+            exit 1
+          fi
+          
+      - name: Generate Graphs and HTML
+        run: asv publish
+
+      - name: Delete env files
+        run: rm -r .asv/env
+      
+      - name: Push results to results repository
+        # if: github.event_name != 'pull_request'
+        continue-on-error: true
+        uses: cpina/github-action-push-to-another-repository@main
+        env:
+          API_TOKEN_GITHUB: ${{ secrets.DEPLOY_PAT_TEMP}}
+        with:
+          source-directory: .asv
+          destination-github-username: atharva-2001
+          destination-repository-name: tardis-benchmark-test-results
+          user-email: aryaatharva18@gmail.com
+          target-branch: main
+          target-directory: .asv
+      
+      - name: Compare HEAD with master if PR
+        if: github.event_name == 'pull_request'
+        continue-on-error: true # TODO: step failed sporadically while testing
+        run: |
+          asv continuous --show-stderr ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }}  | tee asv-output.log  
+          if grep -q Traceback asv-output.log; then 
+            echo "Some benchmarks have errors!"
+            exit 1
+          fi
+      
+      - name: Output logs for comment
+        id: asv_logs
+        run: |
+          EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
+          echo "text<<$EOF" >> $GITHUB_OUTPUT
+          echo "$(cat asv-output.log)" >> $GITHUB_OUTPUT
+          echo "$EOF" >> $GITHUB_OUTPUT
+
+      - name: Write comment
+        uses: actions/github-script@v6
+        with:
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: `
+              *\*beep\* \*bop\**
+              Hi human, 
```

Comment:
Should comments be made via GitHub Actions Bot or the TARDIS bot? 
I can make it via both. 

GitHub Actions bot example- https://github.com/atharva-2001/tardis-benchmark-test/pull/3#issuecomment-1512406942

Reply:
tardis bot only should be sufficient